typedb/common/rpc/request_builder.py

```diff
@@ -426,9 +420,39 @@ def role_type_get_relation_types_req(label: Label):
     return type_req(req, label)
 
 
-def role_type_get_players_req(label: Label):
+def role_type_get_player_types_req(label: Label):
+    req = concept_proto.Type.Req()
+    req.role_type_get_player_types_req.CopyFrom(concept_proto.RoleType.GetPlayerTypes.Req())
+    return type_req(req, label)
+
+
+def role_type_get_player_types_explicit_req(label: Label):
     req = concept_proto.Type.Req()
-    req.role_type_get_players_req.CopyFrom(concept_proto.RoleType.GetPlayers.Req())
+    req.role_type_get_player_types_explicit_req.CopyFrom(concept_proto.RoleType.GetPlayersTypesExplicit.Req())
```

Comment:
Seems to be correctly named `GetPlayerTypesExplicit` in `typedb-protocol`. Let's do a quick sweep for typos, they don't cause the build to fail in Python.
```suggestion
    req.role_type_get_player_types_explicit_req.CopyFrom(concept_proto.RoleType.GetPlayerTypesExplicit.Req())
```